data/users.json

```diff
@@ -1,18 +1,26 @@
 [
   {
     "id": "1",
-    "name": "Vasya"
+    "name": "vasya",
+    "password": "7815696ecbf1c96e6894b779456d330e",
```

Comment:
Not the best way to store passwords in db.

Reply:
Hmm... But where then?
From HW6 description: 8. Modify application to respond all routes from Homework 4 and return data from the database.

Reply:
I mean it's not secure. Better to store hash and salt.